dlrover/python/diagnosis/common/constants.py

```diff
@@ -19,15 +19,30 @@ class EnvConfigKey(object):
 class InferenceConfigKey(object):
     LOG_FILE = "log_file"
     ERRORS = "errors"
+    LOGS = "logs"
+    RANK = "rank"
+    EXPIRED_TIME_PERIOD = "expired_tie_period"
+    EXECUTABLE_TIME_PERIOD = "executable_time_period"
+
+    EVENT_TYPE = "event_type"
+    EVENT_INSTANCE = "event_instance"
+    EVENT_ACTION = "event_action"
+    EVENT_MSG = "event_msg"
+    EVENT_LABELS = "event_labels"
 
 
 class DiagnosisConstant(object):
     MASTER_DIAGNOSIS_OBSERVING_INTERVAL_SECS = 180
-    AGENT_PERIODICALLY_DIAGNOSIS_INTERVAL_SECS = 60
+    AGENT_PERIODICALLY_DIAGNOSIS_INTERVAL_SECS = 5
```

Comment:
The 5 seconds interval is too frequent. 
If a certain operation requires a shorter interval, differentiation in implementation is necessary.

Reply:
Is my understanding correct?

The fixed interval for the thread is 5 seconds, and then each time problems are retrieved, the time period is checked to control problem diagnostics at different intervals?

Reply:
Yes. The 5 seconds is the min time interval. Then we can configure different problems to be handled with 2 x Interval, 3 x interval, etc.